<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteQueryDialog</class>
 <widget class="QDialog" name="RouteQueryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>路线查询</string>
  </property>
  <widget class="QPushButton" name="queryButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>89</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>路线最短</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>251</width>
     <height>64</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="verticalSpacing">
     <number>16</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>起点站:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="startingStationLine"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>终点站:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="terminalStationLine"/>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>561</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>89</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queryButton_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>89</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>换乘最少</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
